tests/cupy_tests/indexing_tests/test_indexing.py

```diff
@@ -224,3 +226,65 @@ def test_choose_broadcast_fail(self, xp, dtype):
         a = xp.array([0, 1])
         c = testing.shaped_arange((3, 5, 4), xp, dtype)
         return a.choose(c)
+
+
+@testing.gpu
+class TestSelect(unittest.TestCase):
+
+    @testing.for_all_dtypes(no_bool=True, no_complex=True)
```

Comment:
Please add tests for invalid parameters, empty lists and different sized list.

Reply:
Done